cypress/e2e/michael.cy.ts

```diff
@@ -0,0 +1,16 @@
+describe('drag', () => {
```

Comment:
`drag` is for a specific test case. Do not think it applies to multiple test cases in this file / descirbe.